### PR TITLE
installed styled-components for dynamically colored labels

### PR DESCRIPTION
**Pull request by trblackw**
title's pretty self-explanatory.

### CODE REVIEW

File: src/features/Issues/Issues.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ import "./Issues.scss";
 import { Link } from "react-router-dom";
 import { connect } from "react-redux";
 import { getIssues, filterIssues } from "./actions";
-import Icon from "./components/Icon/Icon";
+import Icon from "../Icon/Icon";
```
Comment (dillionverma) [inline src/features/Issues/Issues.js]: Woops, should be the other import instead

Comment (dillionverma) [reply src/features/Issues/Issues.js]: image

**Comment by trblackw** [comment]
I just used the most recent one I saw locally. My b

**Comment by dillionverma** [comment]
No worries, looks good otherwise!

**Comment by trblackw** [comment]
👍🏼👍🏼 Let me know what else you’re trying to accomplish with this and
I’ll see if I can help

**Comment by dillionverma** [comment]
Definitely. This was just meant to be a mini project for hacktoberfest. Wanted to see what we could accomplish together as a team 😄 I will add more issues soon.

Once you change the import I will merge this PR in

**Comment by trblackw** [comment]
hey man bit of a noob question, I don't do a lot of PRs, I've fixed the
import locally and pushed it to my forked repo of the project, do I open
another PR to fix the conflict? I feel like no, but I'm not sure what to do
currently to fix it

**Comment by trblackw** [comment]
also, just so I'm clear, which file path am I supposed to include? "
./components/Icon/Icon" ?

**Comment by dillionverma** [comment]
Hmm seems like I'm having troubles too lol. For some reason its not giving me options on cli either

**Comment by dillionverma** [comment]
Check #35

**Pull request closed without merging**